modules/preparing-an-inital-cluster-deployment-for-mce-disconnected.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// * installing_with_agent_based_installer/preparing-an-agent-based-installed-cluster-for-mce.adoc
+
+:_content-type: PROCEDURE
+[id="preparing-an-inital-cluster-deployment-for-mce-disconnected_{context}"]
+
+= Preparing an initial cluster deployment for multicluster engine for Kubernetes(MCE) Operator while disconnected
```

Comment:
```suggestion
= Preparing to install multicluster engine for Kubernetes (MCE) Operator while disconnected
```
Is there a way you can shorten this heading? Just a suggestion, but it seems a bit long to me.

Reply:
Yes I agree its long but I could not find any words I would like to remove